### DISCUSSION TITLE
Mac电脑本地部署alist，局域网内其他设备无法访问

### DISCUSSION THREAD

**Discussion by hengzi-hub** [AlistGo/alist#3949] [Q&A]
最近碰到一个问题，不知道怎么解决。描述如下：
使用 alist安装部署在 Mac 电脑上，然后挂载阿里云盘，通过 Mac本地的` ip:5244`能访问到alist页面，Mac本地的infuse也能正常读取到阿里云盘里的数据；
但是在同一个局域网下的其他设备上(iPhone、Apple TV) 的infuse却读取不了相同的webdav配置，所以我在想使用 alist部署在 Mac电脑上的方法是不是只能在Mac本地观看，其他设备就不行了，即使是同一个局域网也不行？
各位有遇到过同样的问题么？如果有的话，是怎么解决的呢？[难过]

**Comment by sgy0312** [comment]
蹲一个
